oci-unit-tests/helper/test_helper.sh

```diff
@@ -151,15 +158,28 @@ check_manifest_exists()
         for got_file in ${files}; do
             if [ "${got_file}" = "${want_file}" ]; then
                 debug "found ${got_file}"
-                found=1
+                if [ -n "${found}" ]; then
+                    found="${found} ${got_file}"
+                else
+                    found="${got_file}"
+                fi
             fi
         done
     done
 
-    if [ "${found}" -eq 0 ]; then
+    if [ -z "${found}" ]; then
         echo "E: No manifest file found in the image" > /dev/stderr
         debug "not found"
         return 1
+    else
+      debug "Verifying required manifest file(s) presence [${required_manifest_files}]"
+        for required_file in ${required_manifest_files}; do
+            if ! echo "${found}" | grep -q -w "${required_file}"; then
```

Comment:
I'd prefer `grep -qFw` just because we know `${required_file}` is never going to be a regexp.